docs/Local Development Install.md

```diff
@@ -0,0 +1,181 @@
+
+1. [DCMFOSS Local Development Deployment](#dcmfoss-local-development-deployment)
+   - [Prerequisites](#prerequisites)
+   - [Guide](#guide)
+      - [Cloning the repository](#cloning-the-repository)
+      - [Open in your own IDE of choice](#open-in-your-own-ide-of-choice)
+      - [Docker desktop](#docker-desktop)
+      - [Running containers](#running-containers)
+      - [Running project first time for init configs](#running-project-first-time-for-init-configs)
+      - [Running Keycloak](#running-keycloak)
+      - [Fetching the keycloak client credential](#fetching-the-keycloak-client-credential)
+      - [Configure postman collection requests](#configure-postman-collection-requests)
+      - [Run the front-end](#run-the-front-end)
+   - [Postmand Collection](#Postman-collection)
+   - [Notice](#notice)
+
+
+# DCMFOSS Local Development Deployment
+
+
+## Prerequisites
+
+This guide assumes the following prerequisites are true.
+It was built considering an updated windows build with docker running, but will try to provide general instructions.
+
+  - Basic coding knowledge
+  - Basic Docker/Kubernetes knowledge
+  - JDK 17 ( or Higher)
+  - Maven 'Spring Boot is compatible with Apache Maven 3.5 or above'
+  - Git/Github Desktop
+  - NodeJS installed
+  - Yarn installed
+  - Docker Desktop
+
+
+## Guide:
+Let's begin local development install!
+
+- ### Cloning the repository
+    Clone the repository from the url below
+
+      git clone https://github.com/catenax-ng/tx-demand-capacity-mgmt.git
+  
+    ![Using Github desktop](images/dev/1.png "Cloning the repo")
+
+- ### Open in your own IDE of choice
+    Boot your IDE of choice and open the backend repo,
+    add the specification as a module of the backend project.
+    
+    ![IntelliJ IDE](images/dev/2.png "resolving dependencies")
+
+    ![IntelliJ IDE modules](images/dev/2_5.png "resolving dependencies")
+
+  it is normal for startup to have errors, let your IDE finish configuring and download dependencies.
+
+  go ahead and run a maven clean and install on the specification cycle.
+  if it fails saying you don't have a JDK, choose JDK17 of your choice.
+
+- ### Docker desktop
+
+    Before you run the project we need to setup Postgres and Keycloak.
+    Let's begin with installing docker(if you are running windows make sure to use WSL2).
+    After installing docker on your machine lets install postgres.
+    if you get an error of Hyper V make sure **virtualization is enabled on your motherboard
+    and if you're on windows, enable Hyper-V feature-**
+    if you get an error of wsl update not supported
+    download and install 
+
+    https://wslstorestorage.blob.core.windows.net/wslblob/wsl_update_x64.msi 
+    
+    open a powershell and type 
+
+      wsl --set-default-version 2
+    
+    then try again
+
+    ![Docker Desktop](images/dev/3.png "Docker install")
+
+- ### Running containers
+
+  with docker running open a command terminal
+
+      docker pull postgres
```

Comment:
Version?

Reply:
Docker Desktop Version 4.27.1 

@Ruskyy @Bailonis 
Please tell me which postgres version is supposed to be pulled exactly. I assume 16.1